app/views/commontator/comments/create.js.erb

```diff
@@ -29,10 +29,6 @@
   ) %>");
 <% end %>
 
-<% if @update_icon %>
-$('#commentsIcon').addClass('new-comment');
-<% end %>
```

Comment:
Where was the `@update_icon` variable declared beforehand? I cannot find any occurrences, neither in our codebase, nor in that of `commontator`.

Reply:
I removed the definition of @update_icon from the comments_controller.